lib/p5.sound.min.js

```diff
@@ -1,3 +1,3 @@
prototype.update=function(t){var e=this.energy=t.getEnergy(this.f1,this.f2)/255;e>this.cutoff&&e>this.threshold&&0<e-this.penergy?(this._onPeak(),this.isDetected=!0,this.cutoff=e*this.cutoffMult,this.framesSinceLastPeak=0):(this.isDetected=!1,this.framesSinceLastPeak<=this.framesPerPeak?this.framesSinceLastPeak++:(this.cutoff*=this.decayRate,this.cutoff=Math.max(this.cutoff,this.threshold))),this.currentValue=e,this.penergy=e},p5.PeakDetect.prototype.onPeak=function(t,e){var n=this;n._onPeak=function(){t(n.energy,e)}}}.call(e,n,e,t))||(t.exports=i)},function(t,e,n){"use strict";var i;void 0===(i=function(t){var r=n(1);p5.Gain=function(){this.ac=r.audiocontext,this.input=this.ac.createGain(),this.output=this.ac.createGain(),this.input.gain.value=.5,this.input.connect(this.output),r.soundArray.push(this)},p5.Gain.prototype.setInput=function(t){t.connect(this.input)},p5.Gain.prototype.connect=function(t){var e=t||p5.soundOut.input;this.output.connect(e.input?e.input:e)},p5.Gain.prototype.disconnect=function(){this.output&&this.output.disconnect()},p5.Gain.prototype.amp=function(t){var e=1<arguments.length&&void 0!==arguments[1]?arguments[1]:0,n=2<arguments.length&&void 0!==arguments[2]?arguments[2]:0,i=r.audiocontext.currentTime,o=this.output.gain.value;this.output.gain.cancelScheduledValues(i),this.output.gain.linearRampToValueAtTime(o,i+n),this.output.gain.linearRampToValueAtTime(t,i+n+e)},p5.Gain.prototype.dispose=function(){var t=r.soundArray.indexOf(this);r.soundArray.splice(t,1),this.output&&(this.output.disconnect(),delete this.output),this.input&&(this.input.disconnect(),delete this.input)}}.call(e,n,e,t))||(t.exports=i)},function(t,e,n){"use strict";var i;void 0===(i=function(t){var i=n(5);function o(t){for(var e,n="number"==typeof t?t:50,i=new Float32Array(44100),o=Math.PI/180,r=0;r<44100;++r)e=2*r/44100-1,i[r]=(3+n)*e*20*o/(Math.PI+n*Math.abs(e));return i}p5.Distortion=function(t,e){if(i.call(this),void 0===t&&(t=.25),"number"!=typeof t)throw new Error("amount must be a number");if(void 0===e&&(e="2x"),"string"!=typeof e)throw new Error("oversample must be a String");var n=p5.prototype.map(t,0,1,0,2e3);this.waveShaperNode=this.ac.createWaveShaper(),this.amount=n,this.waveShaperNode.curve=o(n),this.waveShaperNode.oversample=e,this.input.connect(this.waveShaperNode),this.waveShaperNode.connect(this.wet)},p5.Distortion.prototype=Object.create(i.prototype),p5.Distortion.prototype.process=function(t,e,n){t.connect(this.input),this.set(e,n)},p5.Distortion.prototype.set=function(t,e){if(t){var n=p5.prototype.map(t,0,1,0,2e3);this.amount=n,this.waveShaperNode.curve=o(n)}e&&(this.waveShaperNode.oversample=e)},p5.Distortion.prototype.getAmount=function(){return this.amount},p5.Distortion.prototype.getOversample=function(){return this.waveShaperNode.oversample},p5.Distortion.prototype.dispose=function(){i.prototype.dispose.apply(this),this.waveShaperNode&&(this.waveShaperNode.disconnect(),this.waveShaperNode=null)}}.call(e,n,e,t))||(t.exports=i)}]);
```

Comment:
The compiled versions of the library are still missing a lot of code, any idea why that is?

In general I'd recommend excluding `lib` files from PR's until we're ready to bump the library version number. But, if you want to include, make sure they compiled successfully !

Reply:
@therewasaguy  
i am not focusing on lib/ in my latest commits , and if you are checking it after my latest commit , build the file once again , i dont think the file is shrunk , it is 443kb as seen on linux